tests/bulk_create/tests.py

```diff
@@ -217,3 +217,16 @@ def test_set_pk_and_query_efficiency(self):
         self.assertEqual(Country.objects.get(pk=countries[1].pk), countries[1])
         self.assertEqual(Country.objects.get(pk=countries[2].pk), countries[2])
         self.assertEqual(Country.objects.get(pk=countries[3].pk), countries[3])
+
+    @skipUnlessDBFeature('can_return_ids_from_bulk_insert')
```

Comment:
Should we also add a testcase that if the database does not have the feature, the `.state` field is not updated?